content/guides/component-lifecycle.mdx

```diff
@@ -6,9 +6,10 @@ These milestones summarize the maturity lifecycle of UI components in Primer. Co
 
 ## Experimental
 
-Proof-of-concept, often built outside of Primer.
+The component exists as a proof-of-concept or is work in progress.
 
-- The component exists as a proof-of-concept. In many cases experimental components may be built outside of Primer, often in a GitHub application such as GitHub.com.
+- The component can be on any state on the maturity spectum from low to high. Please refer to the `status checklist in the component documentation` for fine-grained information about the component's maturity.
```

Comment:
We don't seem to have status checklist on `view-components` or maybe I missed that. I am not sure how inclusive this criteria 🤔 @primer/rails-reviewers

Reply:
Removed the bullet point so not relevant anymore.